content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -319,21 +319,24 @@ Some values of an object are typically generated before the object is persisted.
 
 ## Server Side Apply
 
-Server Side Apply (SSA) is offering an apply path on the apiserver side of kubernetes instead of the previous/current Client Side Apply (CSA) that happens in kubectl.
+{{< feature-state for_k8s_version="v1.14" state="alpha" >}} In vesion 1.14, if the Server Side Apply feature is enabled, The `PATCH` endpoint will accept the additional `application/apply-patch+yaml` content type. Server Side Apply allows clients other than kubectl to perform the Apply operation, and will eventually fully replace the complicated Client Side Apply logic that only exists in kubectl.
 
-*Note:* This is an alpha feature and needs to be enabled by setting the related (Feature Gate)[docs/reference/command-line-tools-reference/feature-gates.md].
+Compared to the `last-applied` annotation managed by `kubectl`, Server Side Apply uses a more declarative approach, which tracks a user's field management, rather than a user's last applied state. This means that as a side effect of using Server Side Apply, information about which field manager manages each field in an object becomes available.
```

Comment:
I don't know if I'd compare it directly to what we had before? Though it might be useful for some people

Reply:
Assuming we start the section with

> Server Side Apply allows clients other than kubectl to perform the Apply operation, and will eventually fully replace the complicated Client Side Apply logic that only exists in kubectl.

then it makes sense to me to say how our approach is conceptually different